spec/lib/miq_automation_engine/miq_ae_object_spec.rb

```diff
@@ -1,3 +1,5 @@
+include AutomationSpecHelper
```

Comment:
This should not be included at the global level.  If you need specific things to be shared with both automation specs and with lib/miq_automation_engine, we should come up with a new mixin.